cn/docs/tutorials/kubernetes-basics/cluster-intro.html

```diff
@@ -31,12 +31,12 @@
 			<div class="col-md-8">
 				<h3>Kubernetes 集群</h3>
 				<p>
-				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 中的抽象允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 以更有效的方式自动化、跨集群的容器应用程序的分发和调度。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，为生产环境准备的。
+				<b>Kubernetes 用于协调高度可用的计算机集群，这些计算机群集被连接作为单个单元工作。</b> Kubernetes 的抽象性允许您将容器化的应用程序部署到集群，而不必专门将其绑定到单个计算机。为了利用这种新的部署模型，应用程序需要以将它们与各个主机分离的方式打包: 它们需要被容器化。容器化应用程序比过去的部署模型更灵活和可用，其中应用程序直接安装到特定机器上，作为深入集成到主机中的软件包。 <b>Kubernetes 在一个集群上以更有效的方式自动分发和调度容器应用程序。</b> Kubernetes 是一个 <a href="https://github.com/kubernetes/kubernetes">开源</a> 平台，并且已经准备好了帮助生产。
 				</p>
 				<p>Kubernetes 集群由两种类型的资源组成:
 					<ul>
-			            <li>一个 <b>Master</b> 调度节点</li>
-			            <li><b>Nodes</b> 应用程序实际运行的地方</li>
+			            <li>一个 <b>Master</b> 调度集群</li>
+			            <li><b>节点</b> 是应用程序实际运行的地方</li>
```

Comment:
same comments here, can we both keep the English and Chinese for specific technical words here, like Nodes